app/helpers/govspeak_helper.rb

```diff
@@ -253,7 +253,7 @@ def build_govspeak_document(govspeak, images = [])
 
   class AssetHostDecorator < SimpleDelegator
     def url(*args)
-      (Whitehall.asset_root || "") + super(*args)
+      (Whitehall.public_asset_host || "") + super(*args)
```

Comment:
`super` bare would pass along the args. In fact, you could even change the method signature here to `def url(*)` and this would still do the trick.